pages/cheggblog.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="/stylesheets/style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About Me</title>
+</head>
+<body>
+    <header></header>
+    <nav>
+      <h1>Emily Saeli</h1>
+      <a href="index.html">Home</a>
+      <a href="about.html">About Me</a>
+      <a href="portfolio.html">Code Portfolio</a>
+      <a href="debate.html">Debate</a>
+      <a href="publications.html">Publications</a>
+      <a href="workexperience.html">Resume</a>
+    </nav>
+<p>
+The original article can’t be found on Chegg’s website anymore but others I have written can (and should) be found below: </p>
+
+<ul>
+    <li><a href="https://www.chegg.com/play/student-life/freshman-real-talk/the-unspoken-rules-of-college/">The Unspoken Rules of College</a></li>
+    <li><a href="https://www.chegg.com/play/student-life/freshman-real-talk/survive-move-in-day/">How to Survive Move-in Day</a></li>
+    <li><a href="https://www.chegg.com/play/student-life/freshman-real-talk/senior-real-talk-which-major-to-pick/">Which Major to Pick</a></li>
+    <li><a href="https://www.chegg.com/play/student-life/freshman-real-talk/senior-real-talk-anxiety-in-college/">Anxiety in College</a></li>
+</ul>
+
+
+
+
+
+
+<h2>SENIOR REAL TALK: WRITING TIPS</h2>
+<h4>by Emily Saeli</h4>  
```

Comment:
Style: You shouldn't skip heading levels.  If you want this text to be smaller just use CSS.  (A `h3` here would be fine, as would just having a specially styled paragraph or span or something.)